app/src/main/java/com/android/streetworkapp/ui/profile/AddFriend.kt

```diff
@@ -39,7 +35,8 @@ import com.android.streetworkapp.ui.navigation.NavigationActions
 @Composable
 fun AddFriendScreen(
     navigationActions: NavigationActions,
-    userViewModel: UserViewModel = viewModel(factory = UserViewModel.Factory)
+    userViewModel: UserViewModel = viewModel(factory = UserViewModel.Factory),
+    innerPaddingValues: PaddingValues = PaddingValues(0.dp)
```

Comment:
Question: same question as the previous padding problem what's the use case ?

Reply:
you're supposed to pass it as a param when using scaffold. Haven't tested personnally but a non zero value should be passed when bottom bar is set, otherwise some content could be hidden